examples/sec-sentiment-analysis/requirements.in

```diff
@@ -0,0 +1,8 @@
+jupyter
+torch
+torchaudio
+torchvision
+transformers
+ratelimit
+requests
+unstructured>=0.2.1
```

Comment:
Do we need this as a requirement now that it's in the same repo?

Reply:
Only reason I still like that in there is that it lets you `pip install -r requirements.txt` and run the example notebooks if you got linked directly to the example directory. So it could be helpful if you don't have `unstructured` install locally or if you don't have the repo added to your PYTHONPATH.